src/components/editions/video/index.tsx

```diff
@@ -0,0 +1,45 @@
+// ----- Imports ----- //
+import { css } from '@emotion/react';
+import type { FC } from 'react';
+
+// ----- Styles ----- //
+
+const videoWrapperStyles = css`
+	width: 100%;
+	position: relative;
+	padding-bottom: 56.25%;
+`;
+
+const videoStyles = css`
+	position: absolute;
+	top: 0;
+	left: 0;
+	width: 100%;
+	height: 100%;
+`;
+
+// ----- Component ----- //
+
+interface Props {
+	title: string;
+	atomId: string;
+}
+
+const Video: FC<Props> = ({ title, atomId }) => {
+	return (
+		<div css={videoWrapperStyles}>
+			<iframe
+				title={title}
+				css={videoStyles}
+				frameBorder="0"
+				scrolling="no"
+				allowFullScreen
+				src={`https://embed.theguardian.com/embed/atom/media/${atomId}#noadsaf`}
```

Comment:
Question for @paperboyo and @akash1810, is this the correct URL to use for embedding media atoms?